src/files.js

```diff
@@ -160,6 +182,52 @@ module.exports = (common) => {
           })
         })
 
+        it('add a nested dir as array with progress', (done) => {
+          // Needs https://github.com/ipfs/js-ipfs-api/issues/339 to be fixed
+          // for js-ipfs-api + go-ipfs
+          if (!isNode) { return done() }
```

Comment:
Hm, why only non-node environment?

Reply:
hmm, good question - this is basically a copy of the above tests which carries this flag.

Reply:
@VictorBjelkholm check the issue linked in the comments. go-ipfs started error'ing in v0.4.2

@dryajov mind trying running the dir tests (by removing the if clause) against latest go-ipfs and see if this is still an issue?

Reply:
🔴 @dryajov did you get to try this one too?

Reply:
I did, it seems to still be an issue with the latest master.